src/xml_utils.py

```diff
@@ -84,12 +84,12 @@ def read_xml_file(self, full_path):
 
         link = root.findall(".//atom:entry/atom:link", namespaces)[0].get('href')
         if "bitcoin-dev" in link:
-            domain = "https://lists.linuxfoundation.org/pipermail/bitcoin-dev/"
+            domain = "https://gnusha.org/url/https://lists.linuxfoundation.org/pipermail/bitcoin-dev/"
```

Comment:
our replacement script only replaces `url` and `thread_url` of documents, right? which means that this `domain` will never match.

Reply:
Let me fix it!